src/deploy/hosting/prepare.ts

```diff
@@ -30,10 +30,6 @@ export async function addTaggedFunctionsToOnlyString(
   context: Context,
   options: HostingOptions & Options
 ): Promise<boolean> {
-  if (!options.only) {
```

Comment:
This was here because a deployment that doesn't have an `--only` flag doesn't need to update `--only`; it's already deploying everything.

Reply:
only is undefined when deploying to a preview channel, we should remain targeted